src/textual/events.py

```diff
@@ -210,7 +214,7 @@ def is_printable(self) -> bool:
         Returns:
             bool: True if the key is printable. False otherwise.
         """
-        return self.key not in KEY_VALUES
+        return self.key == Keys.Space or self.key not in KEY_VALUES
```

Comment:
`Keys` was being used for non-printable characters only. Now that "space" is inside it, it broke this usage and so I've added this special case to resolve it.